gcp-observability/src/main/java/io/grpc/gcp/observability/GcpObservability.java

```diff
@@ -146,6 +146,26 @@ public void close() {
     }
   }
 
+  /**
+   * Method to close along with sleep time explicitly.
+   *
+   * @param sleepTime sleepTime
+   */
+  public void closeWithSleepTime(long sleepTime) {
```

Comment:
`sleepTime` has no unit attached. You should either include the unit in the same, like `sleepTimeMillis` (this is more common in fields) or pass a `TimeUnit` as another parameter (this is more common in APIs). So here I suggest passing in a `TimeUnit`.